### PR TITLE
LPD-28216

### PR DESCRIPTION
**Pull request by rijema**
The test is almost finished. I'm only waiting another PR to finish before sending together. This PR is the validation to send a fix to the customer.

### CODE REVIEW

**Comment by rijema** [comment]
ci:test:sf

**Comment by rijema** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-task-web/src/main/java/com/liferay/portal/workflow/task/web/internal/portlet/MyWorkflowTaskPortlet.java [hunk 1]
```diff
@@ -134,15 +136,38 @@ protected boolean isSessionErrorException(Throwable throwable) {
...
-		_workflowTaskPermission.check(
-			themeDisplay.getPermissionChecker(), workflowTask, groupId);
+		if (_workflowTaskPermission.contains(
+				themeDisplay.getPermissionChecker(), workflowTask, groupId)) {
+
+			renderRequest.setAttribute(WebKeys.WORKFLOW_TASK, workflowTask);
+			renderRequest.setAttribute(
+				WebKeys.WORKFLOW_TASK_READ_ONLY, Boolean.FALSE);
+
+			return;
```
Comment (linolaoj) [inline modules/apps/portal-workflow/portal-workflow-task-web/src/main/java/com/liferay/portal/workflow/task/web/internal/portlet/MyWorkflowTaskPortlet.java:153]: Hi @rijema, would a `else` be more clear here?

**Review by linolaoj — CHANGES_REQUESTED** [review]

**Comment by rijema** [comment]
ci:test:sf

**Comment by rijema** [comment]
Hey @linolaoj , could you also see the .jsp? Should I create a variable for the boolean workflowTaskDisplayContext.isWorkflowTaskReadOnly()?

**Comment by linolaoj** [comment]
> Hey @linolaoj , could you also see the .jsp? Should I create a variable for the boolean workflowTaskDisplayContext.isWorkflowTaskReadOnly()?

@rijema , it is ok to have it inline. No variable is needed on the jsp for this.

**Comment by rijema** [comment]
ci:test:sf

**Comment by linolaoj** [comment]
ci:test:workflow

**Comment by rijema** [comment]
ci:test:relevant

**Comment by rijema** [comment]
All the tests are failing even before I submit this PR. I'll be fixing the semver.

**Comment by linolaoj** [comment]
@rijema, this should be good to deliver an SDH, I'll talk to you about some source format that we should apply to the final fix.

**Pull request closed without merging**